paywall/src/unlock.js/startup.ts

```diff
@@ -65,6 +66,16 @@ export function startup(
   )
   iframes.init(config)
 
+  const handshake = new Postmate({
+    url: constants.paywallUrl + '/static/data-iframe.2.0.html',
+    classListArray: ['unlock', 'start'],
+  })
+
+  handshake.then(child => {
+    console.log('Data iframe handshake completed!')
+    ;(window as any).dataIframe = child
```

Comment:
This is just here for testing purposes. We have to figure out exactly where this step belongs. However, once this promise resolves, you can go into the console and run:

```javascript
dataIframe.call('initializeReader', {
  lockAddresses: [ /* a list of lock addresses */ ], 
  accountAddress: "", /* your address */
})
```

And the reader will initialize and start querying for relevant data. Then calls to the model will start to bear fruit:

```javascript
dataIframe.get('locks').then(console.log)

dataIframe.get('keys').then(console.log)
```